language/ir-testsuite/tests/move/generics/phantom_params/constraints_abilities_ok.mvir

```diff
@@ -0,0 +1,15 @@
+module M {
+    struct NoAbilities { a: bool }
+    struct HasAbilities<phantom T1, T2> has drop, copy, store, key { a: bool }
+
+    struct S1<T: drop + copy + store + key> { a: bool }
```

Comment:
same as above, good to have the single and multiple cases

Reply:
done